core/src/iam/verify.rs

```diff
@@ -254,9 +234,17 @@ pub async fn token(kvs: &Datastore, session: &mut Session, token: &str) -> Resul
 						#[cfg(not(feature = "jwks"))]
 						_ => return Err(Error::AccessMethodMismatch),
 					}?;
-
 					// Verify the token
 					decode::<Claims>(token, &cf.0, &cf.1)?;
+					// AUTHENTICATE clause
+					if let Some(au) = de.authenticate.clone() {
```

Comment:
(same) Do we have to clone each time we test this condition?
Couldn't use a ref, and clone what we need once the condition is met?

Reply:
Good catch! Addressed in [d4d2a4b](https://github.com/surrealdb/surrealdb/pull/4385/commits/d4d2a4bd995b1bdc0a4d7752961755c6b3c72bf2).